client/components/vertical-nav/index.jsx

```diff
@@ -2,15 +2,17 @@
  * External dependencies
  */
 import React from 'react';
+import style from './style';
+import withStyles from 'isomorphic-style-loader/lib/withStyles';
 
 const VerticalNav = React.createClass( {
 	render() {
 		return (
-			<div className="vertical-nav">
+			<div className={ style.verticalNav }>
```

Comment:
Ah, so it creates an object of selectors it discovers in `style.scss` and camel-cases them as properties?